mysql-test/suite/tokudb.rpl/t/rpl_tokudb_stm_log-master.opt

```diff
@@ -0,0 +1,2 @@
+--skip-external-locking
+--default-storage-engine=MyISAM
```

Comment:
Ahh, I know what happened, I copied test files from rpl_row|stmt_log_innodb test and this one came from here https://github.com/percona/percona-server/blob/5.6/mysql-test/suite/rpl/t/rpl_row_log_innodb-master.opt